bin/pygrb/pygrb_make_offline_workflow

```diff
@@ -0,0 +1,370 @@
+    row.det_band = None 
+    row.det_fluence = None 
+    row.det_fluence_int = None 
+    row.det_name = None 
+    row.det_peak = None 
+    row.det_peak_int = None 
+    row.det_snr = '' 
+    row.email_time = 0 
+    row.event_dec = 0.0 
+    row.event_dec_err = 0.0 
+    row.event_epoch = '' 
+    row.event_err_type = '' 
+    row.event_ra_err = 0.0 
+    row.start_time_ns = 0 
+    row.event_type = '' 
+    row.event_z = 0.0 
+    row.event_z_err = 0.0 
+    row.notice_comments = '' 
+    row.notice_id = '' 
+    row.notice_sequence = '' 
+    row.notice_time = 0 
+    row.notice_type = '' 
+    row.notice_url = '' 
+    row.obs_fov_dec = 0.0 
+    row.obs_fov_dec_width = 0.0 
+    row.obs_fov_ra = 0.0 
+    row.obs_fov_ra_width = 0.0 
+    row.obs_loc_ele = 0.0 
+    row.obs_loc_lat = 0.0 
+    row.obs_loc_long = 0.0 
+    row.ligo_fave_lho = 0.0 
+    row.ligo_fave_llo = 0.0 
+    row.ligo_delay = 0.0 
+    row.event_status = 0 
+    row.event_ra = float(cp.get("workflow", "ra"))
+    row.event_dec = float(cp.get("workflow", "dec"))
+    row.start_time = int(cp.get("workflow", "trigger-time"))
+    row.event_number_gcn = 9999
+    row.event_number_grb = str(cp.get("workflow", "trigger-name"))
+    tbl.extend([row])
+    xml_file_name = "triggerGRB%s.xml" % str(cp.get("workflow",
+                                                    "trigger-name"))
+    xml_file_path = os.path.join(out_dir, xml_file_name)
+    utils.write_filename(xmldoc, xml_file_path)
+    xml_file_url = "file://localhost%s/%s" % (out_dir, xml_file_name)
+    xml_file = _workflow.File(ifos, xml_file_name, sci_seg, file_url=xml_file_url)
+    xml_file.PFN(xml_file.cache_entry.path, site="local")
+    
+    return xml_file
+
+
+def make_cache(input_files, ifos, name, seg, cacheDir, tags=[]):
+    """
+    Place input_files into a cache file.
+    """
+    cache_file = _workflow.File(ifos, name, seg, extension="lcf",
+                                directory=cacheDir, tags=tags)
+    cache_file.PFN(cache_file.cache_entry.path, site="local")
+
+    # Dump output to file
+    fP = open(cache_file.storage_path, "w")
+    for entry in input_files:
+        start = str(int(seg[0]))
+        duration = str(int(abs(seg)))
+        print >> fP, "%s %s %s %s file://localhost%s" \
+            %(ifos, entry.description.upper(), start, duration,
+              entry.storage_path)
+    fP.close()
+    
+    return cache_file
+
+
+###################################
+# Workflow generation starts here #
+###################################
+
+workflow_name = __program__
+logging.basicConfig(format="%(asctime)s:%(levelname)s : %(message)s",
+                    level=logging.INFO)
+
+# Parse command line options and instantiate pycbc workflow object
+parser = argparse.ArgumentParser()
+parser.add_argument("--version", action="version", version=__version__)
+parser.add_argument("-d", "--output-dir", default=None,
+                    help="Path to output directory.")
```

Comment:
I have now added this as an allowed config option, not command line/ See https://github.com/a-r-williamson/pycbc/commit/bb66c396d2bded08e701ca41cdb9db88edca0159